meshtastic/interfaces/ble/compatibility_service.py

```
@@ -0,0 +1,365 @@
+                alive_result = False
+            thread_is_alive = alive_result if isinstance(alive_result, bool) else False
+            if thread_is_alive is False:
+                return False
+        if prefer_non_blocking:
+            if thread_is_alive is False:
+                return False
+            if put_nowait_callback is None:
+                return False
+            try:
+                put_nowait_callback(callback)
+                return True
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)